cli/src/config.rs

```diff
@@ -31,6 +31,8 @@ pub enum ConfigError {
     ConfigReadError(#[from] config::ConfigError),
     #[error("Both {0} and {1} exist. Please consolidate your configs in one of them.")]
     AmbiguousSource(PathBuf, PathBuf),
+    #[error("Configs found in {0}, {1}, and {2}. Please consolidate your configs in one of them.")]
+    AmbiguousSource3(PathBuf, PathBuf, PathBuf),
```

Comment:
Perhaps, these two error variants can be combined as `AmbiguousSource(Vec<PathBuf>)`. The error message could be formatted as something like `"Configs found in multiple locations: {}", .0.iter().map(|path| path.display()).join(", ")` (untested)